css-layout-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>css layout classes</title>
+</head>
+
+<body>
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
```

Comment:
this row should have 3 column one third classes since there are three columns. each box should have one piece of the content inside